### PR TITLE
update Dockerfile to build vsc plugin

### PR DESCRIPTION
**Pull request by Pog3k**
Add node and npm to the docker container..

### CODE REVIEW

**Review by sebastianpfischer — CHANGES_REQUESTED** [review]
Can you specify which parts are for the python and which parts are for the vsc? The idea is to cluster them and if we need to modify anything in the future to know what does it influence.

File: ci/Dockerfile [hunk 1]
```diff
@@ -13,8 +13,16 @@ RUN apt-get update && apt-get install -y \
...
     python3-pip
+    
+RUN curl -sL https://deb.nodesource.com/setup_15.x| bash -
+RUN apt-get -y install nodejs
+RUN npm install -g typescript
+RUN npm install -g --unsafe-perm vsce 
+    
 ENV HOME=/home/kiso
 RUN python3 -m pip install pipenv
 RUN chmod -R 777 ${HOME}
```
Comment (kakooloukia) [inline ci/Dockerfile:28]: Same here:

RUN python3 -m pip install pipenv && chmod -R 777 ${HOME}

**Review by kakooloukia — CHANGES_REQUESTED** [review]

File: ci/Dockerfile [hunk 2]
```diff
@@ -13,8 +13,16 @@ RUN apt-get update && apt-get install -y \
...
+    curl \
+    gnupg \
     python3 \
     python3-pip
+    
+RUN curl -sL https://deb.nodesource.com/setup_15.x| bash -
+RUN apt-get -y install nodejs
+RUN npm install -g typescript
+RUN npm install -g --unsafe-perm vsce 
+
```
[the remark this answers is not in the document]
Comment (Pog3k) [reply ci/Dockerfile]: done

File: ci/Dockerfile (hunk 1 above)
Comment (Pog3k) [reply ci/Dockerfile]: done

**Comment by Pog3k** [comment]
had to add some packages for the unittest. 
unittest will now run in a virtual frame buffer

**Review by sebastianpfischer — APPROVED** [review]

**Review by kakooloukia — APPROVED** [review]

**Pull request merged**